README.md

```diff
@@ -329,6 +329,13 @@ You can create a `.nvmrc` file containing version number in the project root dir
 
 For example, to make nvm default to the latest 5.9 release for the current directory:
 
+.nvmrc file:
```

Comment:
Why does there need to be a bash command for something so simple. That's confusing